internal/js_library/v2/js_library.bzl

```diff
@@ -3,8 +3,15 @@
 
 load("@bazel_skylib//:lib.bzl", "paths")
 
+def _get_path(ctx, file):
+  if not file.short_path.startswith(ctx.label.package):
+    fail("Unable to recover a relative path: %s did not start with %s" % (file.short_path, ctx.label.package))
+
+  path = file.short_path[len(ctx.label.package)+1:]
+  return path
+
 def _write_config(ctx):
-  output = ctx.actions.declare_file(paths.join(ctx.file._babelrc_tmpl.dirname, "_" + ctx.file._babelrc_tmpl.basename))
+  output = ctx.actions.declare_file('%s.babelrc' % ctx.label.name)
```

Comment:
i ran into issues where the templatized babelrc conflicted for different rules. Since we run _write_config for every rule anyways, this output name is just a unique one.